### ISSUE TITLE
Add circuits to site page

### ISSUE THREAD

**Issue by arjenvri** [netbox-community/netbox#12454]
### NetBox version

v3.5.0

### Feature type

Change to existing functionality

### Proposed functionality

Add circuits to the site details page with their name, provider, type, status. Similar as already the ASN, Contacts, Locations are shown.

### Use case

We found that users find it often hard to locate the circuits that belong to a specific site.
When navigating to sites a users have to find the 'related objects' and click on the count to quickly see the circuits in use in a cite.

**Comment by stuntguy3000** [comment]
I tend to disagree as we risk cluttering the page - all the other objects in a site stem from the same "issue", which is mitigated by looking at the object filters when you click through to the circuits page (https://demo.netbox.dev/circuits/circuits/?site_id=22&site_id=6&site_id=8).

![image](https://user-images.githubusercontent.com/1522389/236431329-24e53b73-d93d-4c3d-a484-89a200b76823.png)

**Comment by arjenvri** [comment]
Thank you. We do already have various other lists on that page like contacts, locations, non racked devices. I feel that the amount relations between all different objects in the gui have been growing over time and this will help users find their way more easily.
